crates/benches/binary/README.md

```diff
@@ -38,8 +38,8 @@ With a Chrome browser installed on your system, make sure you're in the `crates/
 directory, build the wasm module, build the binaries, and then run the script:
 ```sh
 cd browser/wasm
-rustup run nightly wasm-pack build --release --target web
-cd ../../binary
+wasm-pack build --target web
```

Comment:
Did you drop `--release` on purpose here?
I'd expect we still want this for running benches.

Reply:
oops, i overlooked, will add